package.json

```diff
@@ -1713,6 +1713,23 @@
 					"when": "commentController =~ /^github-(browse|review)/"
 				}
 			],
+			"editor/context/copy": [
+				{
+					"command": "issue.copyGithubPermalink",
+					"when": "gitHubOpenRepositoryCount != 0",
+					"group": "3_githubPullRequests@0"
+				},
+				{
+					"command": "issue.copyMarkdownGithubPermalink",
+					"when": "gitHubOpenRepositoryCount != 0",
+					"group": "3_githubPullRequests@1"
+				},
+				{
+					"command": "issue.copyGithubHeadLink",
+					"when": "gitHubOpenRepositoryCount != 0",
+					"group": "3_githubPullRequests@2"
+				}
+			],
 			"editor/context/share": [
```

Comment:
Are you not removing the old ones?

Reply:
After discussing with the team and with Kai, I decided to have the commands in both places. I don't want to break current user's model for where to find the commands, but we also want to make the more discoverable to users who think to look for "Share" first.